### PR TITLE
Initial Commit - Adding Freightliner Cascadia Trucks

### PR DESCRIPTION
**Pull request by as-snehagn**
Adding new vehicle - Freightliner Cascadia trucks.

### CODE REVIEW

**Comment by JWhitleyAStuff** [comment]
There are no extra topics/messages that are enabled/disabled on this vehicle type?

**Comment by mlemm99** [comment]
> There are no extra topics/messages that are enabled/disabled on this vehicle type?

New reports include cruise control, engine brake, marker lamp, wiper, sprayer

**Comment by JWhitleyAStuff** [comment]
@as-snehagn - This PR doesn't reflect the changes that @mlemm99 mentioned.

**Comment by JWhitleyAStuff** [comment]
@as-snehagn Any movement on the items that @mlemm mentioned above? I'd be happy answer any questions.

**Comment by as-snehagn** [comment]
@JWhitleyAStuff I have added the additional reports Mike mentioned, it's being tested. Thanks!

File: src/pacmod3_core.cpp [hunk 1]
```diff
@@ -27,6 +27,9 @@ const uint32_t AS::Drivers::PACMod3::ShiftCmdMsg::CAN_ID = 0x128;
 const uint32_t AS::Drivers::PACMod3::SteerCmdMsg::CAN_ID = 0x12C;
 const uint32_t AS::Drivers::PACMod3::TurnSignalCmdMsg::CAN_ID = 0x130;
 const uint32_t AS::Drivers::PACMod3::WiperCmdMsg::CAN_ID = 0x134;
+const uint32_t AS::Drivers::PACMod3::EngineBrakeCmdMsg::CAN_ID = 0x144;
+const uint32_t AS::Drivers::PACMod3::MarkerLampCmdMsg::CAN_ID = 0x148;
+const uint32_t AS::Drivers::PACMod3::SprayerCmdMsg::CAN_ID = 0x138;
```
Comment (JWhitleyAStuff) [inline src/pacmod3_core.cpp]: Please sort these in order of CAN_ID for easy maintenance.

File: src/pacmod3_core.cpp [hunk 2]
```diff
@@ -43,7 +46,9 @@ const uint32_t AS::Drivers::PACMod3::ParkingBrakeRptMsg::CAN_ID = 0x224;
 const uint32_t AS::Drivers::PACMod3::ShiftRptMsg::CAN_ID = 0x228;
 const uint32_t AS::Drivers::PACMod3::SteerRptMsg::CAN_ID = 0x22C;
 const uint32_t AS::Drivers::PACMod3::TurnSignalRptMsg::CAN_ID = 0x230;
-const uint32_t AS::Drivers::PACMod3::WiperRptMsg::CAN_ID = 0x234;
+const uint32_t AS::Drivers::PACMod3::EngineBrakeRptMsg::CAN_ID = 0x244;
+const uint32_t AS::Drivers::PACMod3::MarkerLampRptMsg::CAN_ID = 0x248;
+const uint32_t AS::Drivers::PACMod3::SprayerRptMsg::CAN_ID = 0x238;
```
Comment (JWhitleyAStuff) [inline src/pacmod3_core.cpp]: Same here with sorting. Also, it looks like WiperRptMsg was accidentally removed.

**Comment by JWhitleyAStuff** [comment]
Holding approval for testing results.

**Comment by as-snehagn** [comment]
@JWhitleyAStuff Testing completed. Have all the additional reports and commands available for the Freightliner.

**Pull request merged**